### ISSUE TITLE
MarkerID Elements gets appended to the main html view

### ISSUE THREAD

**Issue by robpal94** [process-engine/bpmn-studio#504]
#### Describe your issue
While using the color picker, on every color that the color picker hovers, a `<markerId>` Element gets added to the html file under 

```
<svg width="100%" height="100%" style="touch-action: none; user-select: none; -webkit-user-drag: none; -webkit-tap-highlight-color: rgba(0, 0, 0, 0);" data-element-id="Collaboration_1cidyxu" class="">
    
    <defs>
```

One appended element looks like this: 
```
<marker id="sequenceflow-end-#672e2e-black-cvjsli4vrr90vq3duhe7zl97g" viewBox="0 0 20 20" refX="11" refY="10" markerWidth="10" markerHeight="10" orient="auto"><path d="M 1 5 L 11 10 L 1 15 Z" style="fill: black; stroke-width: 1px; stroke-linecap: round; stroke-dasharray: 10000, 1; stroke: black;"></path></marker>
```

Since the html that the browser has to render grows, the perfomance gets worse the more you drag around with the color picker. 

![jun-19-2018 13-42-52](https://user-images.githubusercontent.com/38498731/41595660-fcf681a6-73c7-11e8-98ec-eabb8789fd0a.gif)

#### How can others reproduce the issue?
* In a bpmn diagram, click on a sequence flow
* In the property panel, select the color picker
* Move the mouse in the color picker

#### Issue checklist

Please check the boxes in this list after submitting your Issue:

- [x] I've checked if this issue already exists
- [x] I've included all the information that i think is relevant
- [x] I've added logs and/or screenshots (if applicable)
- [x] I've mentioned PRs and issues that relate to this one

**Comment by SteffenKn** [comment]
Some additional information:
- This only happens when changing the color of a sequence flow
- The marker element always gets added when the color of the sequence flow changes. This also occurs when changing the colors via the normal buttons.
- The main problem is that the markers are not deleted when they are no longer needed

**Comment by cmg-dev** [comment]
That's actually a very cool bug :D

**Comment by Paulomart** [comment]
Im not able to reproduce this bug. I think it was fixed by #475.

**Comment by robpal94** [comment]
Yes, this seemed to fix it.
